Don't reset encryption choice when disabling encryption controls.

Without this patch, the mail.identity.default.encryptionpolicy pref
does not work: adding a key to an account always sets its
encryptionpolicy to 0, no matter the desired default.

--- a/chrome/messenger/content/messenger/am-e2e.js      2010-01-01 00:00:00.000000000 +0100
+++ b/messenger/content/messenger/am-e2e.js   2024-11-06 10:13:09.160790399 +0100
@@ -741,9 +741,6 @@
 function enableEncryptionControls(do_enable) {
   gDisableEncryption.disabled = !do_enable;
   gEnableEncryption.disabled = !do_enable;
-  if (!do_enable) {
-    gEncryptionChoices.value = 0;
-  }
   // If we have a certificate or key configured that allows encryption,
   // then we are able to encrypt drafts, too.
   gEncryptDrafts.disabled = !do_enable;
